src/consensus/pbft/libbyz/Network_open.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Copyright (c) 1999 Miguel Castro, Barbara Liskov.
+// Copyright (c) 2000, 2001 Miguel Castro, Rodrigo Rodrigues, Barbara Liskov.
+// Licensed under the MIT license.
+#pragma once
+
+#include "Message.h"
+
+//
+// Network open messages have the following format:
+//
+#pragma pack(push)
+#pragma pack(1)
+struct Network_open_rep : public Message_rep
+{
+  int id; // id of the replica that generated the message.
+  char padding[5];
```

Comment:
Do we need this?

Reply:
This is needed for the standalone mode. We are still running a couple of tests like that. It will be removed soon.